package.json

```diff
@@ -300,6 +300,7 @@
     "@metamask/browser-passworder": "^4.3.0",
     "@metamask/contract-metadata": "^2.5.0",
     "@metamask/controller-utils": "^11.4.0",
+    "@metamask/design-system-tailwind-preset": "npm:@metamask-previews/design-system-tailwind-preset@0.0.0-preview.07ad19d",
```

Comment:
Importing temporary preview build of the design system tailwind preset that will be used to standardize all tailwind configs across the organization and adhere to design system